.github/actions/app-create/action.yml

```diff
@@ -65,12 +82,18 @@ runs:
 
         # Map the requested GUI toolkit to the input that Briefcase expects
         # Default to the input to accommodate arbitrary toolkits installed as plugins
-        case "$(tr '[:upper:]' '[:lower:]' <<< '${{ inputs.framework }}')" in
-          toga    ) GUI_INPUT=1 ;;
-          pyside6 ) GUI_INPUT=2 ;;
-          ppb     ) GUI_INPUT=3 ;;
-          pygame  ) GUI_INPUT=4 ;;
-          *       ) GUI_INPUT=${{ inputs.framework }} ;;
+        GUI_TOOLKIT="${{ inputs.framework }}"
+        [ "${{ inputs.enable-automation }}" != "false" ] && GUI_TOOLKIT="${GUI_TOOLKIT}-auto"
+        case "$(tr '[:upper:]' '[:lower:]' <<< "${GUI_TOOLKIT}")" in
+          toga         ) GUI_INPUT=1 ;;
+          toga-auto    ) GUI_INPUT=8 ;;
+          pyside6      ) GUI_INPUT=2 ;;
+          pyside6-auto ) GUI_INPUT=6 ;;
+          ppb          ) GUI_INPUT=3 ;;
+          ppb-auto     ) GUI_INPUT=5 ;;
+          pygame       ) GUI_INPUT=4 ;;
+          pygame-auto  ) GUI_INPUT=7 ;;
+          *            ) GUI_INPUT=${{ inputs.framework }} ;;
```

Comment:
So - this *works*, but I wonder if there's a better way.

Instead of trying to program STDIN content, how about we expose the wizard options to the command line. Give each question a metadata identifier (could be as simple as the context variable that is being populated), and expose a command line option that allows the end user to specify answers to those questions - e.g., `briefcase new -Q app_name=my_app -Q formal_name="My App" -Q gui_toolkit=toga`. (I'm using `-Q/--question` here, but I'm open to other spelling)

These command-line values override the default values for each question; that means with --no-input, you'll get default values except for the values you override.

In the case of the template question, you'd be looking to provide the key from the dictionary - so the identifier of the plugin.

Not sure if this needs to be a blocker for this PR; but since it's going to need another pair of .github and briefcase paired changes, I don't know if it might be easier to include in this set.

Reply:
Yeah...this is definitely better. I had been trying to avoid doing this because it only improves CI/dev ergonomics slightly. However, the feature isn't really that complicated.

1 - new repeatable cmdline argument that contribs to a dict
2 - check this dict before asking the question
3 - warn about any unused inputs

There is the complication of invalid inputs. Given this is just a dev thing, I'll probably keep it simple and just output a warning and keep going.

Reply:
- https://github.com/beeware/briefcase/pull/1552